File: JavaScript_12.18/1188 - Área Inferior.js
/**
 * @author Josélio de S. C. Júnior <joseliojrx25@gmail.com>
 * @copyright Josélio de S. C. Júnior 2021
*/

const p = require('fs').readFileSync('/dev/stdin', 'utf8');
const l = p.split('\n');

const operator = l.shift();
let x = 5, y = 7;

const sum = Array.from(Array(12), () =>
    Array.from(Array(12), () => +l.shift()))
    .splice(7, 12)
    .map(e => (e = e.slice(x, y), x--, y++, e))
    .flat()
    .reduce((a, b) => a + b);

const total = operator === 'M' ? sum / 30 : sum;

console.log(total.toFixed(1));

File: Lua_5.4/1188 - Área Inferior.lua
--- author: Josélio Júnior (Lunatic Fox) <joseliojrx25@gmail.com>
--- copyright: Josélio Júnior (Lunatic Fox) 2021

local op, t, x, y, sum = io.read(), {}, 5, 8, 0
for i = 1, 12, 1 do
  t[i] = {}
  for j = 1, 12, 1 do t[i][j] = io.read() end
end
for _ = 1, 7, 1 do table.remove(t, 1) end
for i = 1, #t, 1 do
  for k = 1, x, 1 do t[i][k] = 0 end
  for k = 12, y, -1 do t[i][k] = 0 end
  for j = 1, #t[i], 1 do sum = sum + t[i][j] end
  x = x - 1
  y = y + 1
end
local total = op == 'M' and sum / 30 or sum
print(('%.1f'):format(total))
